content/blog/release-v0.10.md

```diff
@@ -0,0 +1,257 @@
++++
+title = "RGB release v0.10"
+date = 2023-04-10
++++
+
+TL;DR
+-----
+
+LNP/BP Standards Association <https://www.lnp-bp.org>, supported by Hojo
+Foundation, Fulgur Ventures, Pandora Prime, Bitfinex and DIBA, is happy to
+announce the release of RGB v0.10 - the next significant milestone in RGB
+protocol <https://rgb.tech> which brings full support to smart contracts to
+Bitcoin and Lightning. This is a result of a great cross-industry long-term
+collaboration between these bitcoin companies and more than four years of
+extensive development work.
+
+RGB v0.10 can be downloaded and installed as described on <https://rgb.tech>
+website, which also contains a number of user and developer guidelines.
+RGB source code can be found on <https://github.com/RGB-WG>
+
+
+Background of RGB
+-----------------
+
+Some of you might remember an announcement of RGB protocol idea back in
+2018 [1]: it was an idea of “colored coins” over the lightning by Giacomo
+Zucco, based on new concepts developed by Peter Todd - client-side-validation
+and single-use-seals.
+
+In 2019 me (Maxim Orlovsky) and Giacomo Zucco has formed an LNP/BP Standard
+Association, aiming bring RGB from the concept to production. The initiative
+was supported by Bitfinex and Fulgur Ventures. My goal with RGB was not just
+enable assets on Lightning, but of a much larger scope: to build a
+programmability layer for Bitcoin and Lightning, which may unlock much more
+than just tokens: DAOs, decentralized identities and other cases lacked in
+bitcoin itself. This has taken much longer than was expected, and me &
+LNP/BP Standards Association had went through very complex times on this
+road, relying on self-financing for more than a year…
```

Comment:
In 2019 me (Maxim Orlovsky) and Giacomo Zucco formed the LNP/BP Standard
 Association, aiming to bring RGB from the concept to production. The initiative
 was supported by Bitfinex and Fulgur Ventures. My goal with RGB was not just to 
 enable assets on Lightning, but that of a much larger scope: to build a
 programmability layer for Bitcoin and Lightning, which may unlock much more
 than just tokens: DAOs, decentralized identities and other cases that
 bitcoin itself was lacking. This took much longer than was expected, and both myself and the
 LNP/BP Standards Association had gone through very turbulent times on this
 road, relying on self-financing for more than a year…